client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -393,6 +400,19 @@ function mediaButton( editor ) {
 		}
 	} );
 
+	function reselectImage() {
+		// Re-select image node
```

Comment:
I think we can remove this comment since the function is named aptly. If this needs further explication we can add jsDoc to the function.